### PR TITLE
Update swig-email-templates. Solve [#21362]

### PR DESCRIPTION
**Pull request by micksatana**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
       [Change since this commit](https://github.com/andrewrk/swig-email-templates/commit/dae24d8d74d1bab81888781df4a59790606601cf#diff-168726dbe96b3ce427e7fedce31bb0bc)
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by MrHen — APPROVED** [review]

**Comment by RyanCavanaugh** [comment]
Approved by a listed owner. PR ready to merge pending express review by a maintainer.

**Comment by DanielRosenwasser** [comment]
Would you be able to remove the disabled lint rules? Things you might need to adjust include

1. Adding an `X.Y` version number to the header.
2. Use a more precise type than `Function`.

**Comment by DanielRosenwasser** [comment]
Thank you!

**Pull request merged**
